# see message NavORB

uint8 gnssId # GNSS ID
uint8 svId # Satellite ID
uint8 svFlag # Information Flags
uint8 eph # Ephemeris data
uint8 alm # Almanac data
uint8 otherOrb # Other orbit data available
